internal/confmapprovider/discovery/bundle/bundle.d/receivers/rabbitmq.discovery.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{ receiver "rabbitmq" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)rabbitmq.*"}) and not (command matches "splunk.discovery")
```

Comment:
I'm assuming that `name` is the name of the container, and we included in the criteria because it can be a custom image that actually hosts RabbitMQ, is that correct?

Reply:
correct, we are trying to match against 'name'